proto/scene-format.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package turbocow;
+
+message Scene {
+	enum MaterialType {
+		SOLID = 0;
+		GLASS = 1;
+		// other matherials
+	}
+	message Vector3 {
+		float x = 2;
+		float y = 3;
+		float z = 4;
+	}
+	message Transform {
+		int32 id = 5;
+		Vector3 translation = 6;
+		Vector3 rotation = 7;
+		Vector3 scale = 8;
+		int32 parent_id = 9;
+	}
+	message Cube {
+		int32 id = 34;
+		Vector3 upper_bounds = 10;
+		Vector3 lower_bounds = 11;
+		Transform transform = 11;
+	}
+	message Sphere {
+		int32 id = 35;
+		Vector3 positon = 12;
```

Comment:
It seems a bit inconsistent that Cube and Sphere are both primitives, but one has a `transform` field while the other does not.

Reply:
The same about `material`. Why meshed object has material but sphere and cube do not?

Reply:
> It seems a bit inconsistent that Cube and Sphere are both primitives, but one has a `transform` field while the other does not.

I think it would be redundant to force users to specify a transformation matrix even if the object does not need it. See my upper comment for Brokokko.

Reply:
The only parameter of the sphere should be a radius. The position in the space should be defined by transformations. For example, if you want to place a cube and place some spheres around it with a step of 30 degrees at distance 5 meters, it will be almost impossible or require some complicated calculations